cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -235,4 +235,11 @@ export default {
     SearchResults.clickOnCell(barcode, 0);
     ItemRecordView.waitLoading();
   },
+
+  searchByBilled() {
+    cy.do([
+      Accordion({ id: 'fee' }).clickHeader(),
+      Checkbox({ id: 'clickable-filter-fee-billed' }).click(),
+    ]);
```

Comment:
use this method setFilterOptionFromAccordion

Reply:
done